generators/server/templates/_pom.xml

```diff
@@ -1287,7 +1287,7 @@
                         <artifactId>maven-war-plugin</artifactId>
                         <configuration>
                             <%_ if (!skipClient) { _%>
-                            <warSourceDirectory><%= CLIENT_MAIN_SRC_DIR %></warSourceDirectory>
+                            <warSourceDirectory><%= CLIENT_DIST_DIR %></warSourceDirectory>
```

Comment:
Im on phone so cant check, but this should be changed only when client side is not angular1 otherwise this is valid